eos/utils/options.cc

```diff
@@ -386,4 +387,75 @@ namespace eos
     {
         return _value;
     }
+
+    IsospinOption::IsospinOption(const Options & options, const std::vector<OptionSpecification> & specifications, const std::string & key) :
+        SpecifiedOption(options, specifications, key)
+    {
+    }
+
+    IsospinOption::~IsospinOption() = default;
+
+    uint8_t
+    IsospinOption::value() const
+    {
+        static const std::map<std::string, uint8_t> map
+        {
+            { "0",          1     },
+            { "1",          2     },
+            { "0 & 1",      3     },
```

Comment:
Hm, if we want to use this as part of an option's value, we need to avoid the space here. Can we use ``0&1`` instead? Or better (using the above idea of an isospin enum class) ``0|1``?

Also, we might want to specialise ``destringify<Isospin>``.

Reply:
Cf #1001